tslint.full.json

```diff
@@ -1,21 +1,17 @@
 {
-    "$schema": "http://json.schemastore.org/tslint",
-    "extends": [
-      "./tslint.json"
-    ],
-    // This configuration files enabled rules which require type checking
```

Comment:
The changes in this file look like pure whitespace and formatting updates. Can you revert them please, and perhaps send in a new PR?